### DISCUSSION TITLE
2024-02-21 12:21:13.598 [info] [auth] Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by vignesh-gupta** [community/community#109425] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Copilot is still not working for last 3 days: 
Log:
 ```text
2024-02-21 12:21:13.598 [info] [auth] Invalid copilot token: missing token: 403 
2024-02-21 12:21:13.602 [error] [default] Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as vignesh-gupta."
```

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by vignesh-gupta** [reply]
Yeah works now!

**Comment by IslamYH** [comment]
i have the same issue here , and i tried the same solution provided but i still have the same issue

**Reply by Ragbot-18** [reply]
Tried the method, it didn't work for me

**Reply by davecheney** [reply]
@Ragbot-18 your copilot coupon has expired.

@IslamYH your free student coupon is active. The issue is in the configuration of your editor. Please make sure you applied all the updates for VSCode and the Copilot extension.
